src/styles/index.scss

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
 }
+
+.stopwatch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch__container {
+  box-sizing: content-box;
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.stopwatch__center {
+  background-color: #f6a603;
+  height: 5vmin;
+  width: 5vmin;
+  border-radius: 50%;
+  z-index: 1;
+}
+
+.stopwatch__mimutes {
+  height: 20vmin;
+  width: 3vmin;
+  background-color: #0700ff;
+  position: absolute;
+  bottom: 50%;
+  animation: rotate 3600s linear infinite;
+  transform-origin: bottom;
+}
+
+.stopwatch__mimutes--speed-up {
```

Comment:
The typo in the class name `stopwatch__mimutes--speed-up` should be corrected to `stopwatch__minutes--speed-up` to ensure consistency with the HTML and intended functionality.